### ISSUE TITLE
Dark Mode intro page number text-color should be lighter, it's difficult to understand

### ISSUE THREAD

**Issue by rajatkantinandi** [freeCodeCamp/freeCodeCamp#17501]
#### Describe your problem and - if possible - how to reproduce it
**Dark Mode** intro page number **text-color** should be lighter for better **accessibility**, it's difficult to understand the **dark green** text in **dark background**.
A **lighter color**, probably **lime** or something should be used.

#### If possible, add a screenshot here

![screenshot_20180608_230601](https://user-images.githubusercontent.com/17743895/41172337-35b56b42-6b71-11e8-91f9-47dea33687bb.png)
